matchms/Scores.py

```diff
@@ -39,9 +39,9 @@ class Scores:
         similarity_measure = CosineGreedy()
         scores = calculate_scores(references, queries, similarity_measure)
 
-        for (reference, query, score, n_matching) in scores:
+        for (reference, query, score) in scores:
             print(f"Cosine score between {reference.get('id')} and {query.get('id')}" +
-                  f" is {score:.2f} with {n_matching} matched peaks")
+                  f" is {score['score']:.2f} with {score['matches']} matched peaks")
 
     Should output
 
```

Comment:
Line 95-97 should be updated, as there is no longer a similarity function which returns a tuple.